src/dune

```diff
@@ -28,8 +28,12 @@
  ; (promote (until-clean))
 
  (preprocess
+  ; The `pps` field is a list of ppx to apply to the sources before generating JavaScript. melange.ppx allows to use Melange attributes [@mel. ...] (https://melange.re/v2.0.0/communicate-with-javascript/#attributes)
+  ; reason-react-ppx allows to use JSX for ReasonReact components
   (pps melange.ppx reason-react-ppx))
 
+ ;; module_systems is a list of module systems to generate code for (es6, commonjs)
+ ;; Change to (module_systems es6 commonjs) if you want to generate both
```

Comment:
We could link to either https://melange.re/v2.0.0/build-system/#commonjs-or-es6-modules or https://dune.readthedocs.io/en/latest/melange.html#melange-emit.